data/net/minecraft/world/level/block/entity/BellBlockEntity.mapping

```diff
@@ -3,11 +3,26 @@ CLASS net/minecraft/world/level/block/entity/BellBlockEntity
 		COMMENT Warmup counter until raiders are revealed by glow and particle
 	FIELD ticks I
 		COMMENT How long the bell has been ringing
+	METHOD <init> (Lnet/minecraft/core/BlockPos;Lnet/minecraft/world/level/block/state/BlockState;)V
+		ARG 1 worldPosition
```

Comment:
```suggestion
		ARG 1 pos
```
Despite the fact that in `BlockEntity` (and seemingly *only* in `BlockEntity`), the field this is eventually assigned to is called `worldPosition`, I'd advocate for keeping consistency with other `pos` parameters in this call path.

Some other notes:
- We currently map `BlockEntity.<init>` as `worldPosition` [here](https://github.com/ParchmentMC/Parchment/blob/3df8090b42497d2779dee6ff0850f4ae8322193d/data/net/minecraft/world/level/block/entity/BlockEntity.mapping#L4).
- The accessor for `BlockEntity.worldPosition` is called `getBlockPos()`, not `getWorldPosition()`. Meaning from the public API (constructor and accessor), you never see the name `worldPosition`.

cc @ParchmentMC/mapping-standards

Reply:
I'd be fine with mapping this specific parameter in the `BlockEntity` and subclasses' constructors as `pos`, for consistency.

Reply:
I think I already committed the suggestion earlier.